src/genie/libs/parser/iosxe/tests/ShowApConfigGeneral/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,115 @@
+expected_output = {
+    "ap_config_general_info": {
```

Comment:
`ap_name` is better for the value `bg-1-cap1`.
```
"ap_name": {
    "bg-1-cap1": {

Reply:
Updated